nvdaHelper/archBuild_sconscript

```diff
@@ -73,9 +73,9 @@ def COMProxyDllBuilder(env,target,source,proxyClsid):
 	return proxyDll
 env.AddMethod(COMProxyDllBuilder,'COMProxyDll')
 
-# We only support compiling with MSVC 14.2 (2019)
-if not env.get('MSVC_VERSION')=='14.2':
-	raise RuntimeError("Visual C++ 14.2 (Visual Studio 2019) not found")
+# We only support compiling with MSVC 14.2 (2019) or newer
+if not env.get('MSVC_VERSION') >= '14.2':

```

Comment:
I'm not sure this check will do what you think if there happened to be a 14.10, though I do realize this is very unlikely.

Reply:
I guess we could fix this.

```suggestion
if not env.get('MSVC_VERSION') or env.get('MSVC_VERSION').split(".") < (14, 2):
```